hollow/src/main/java/com/netflix/hollow/api/producer/AbstractHollowProducer.java

```diff
@@ -434,6 +441,8 @@ long runCycle(
             throw new RuntimeException(th);
         } finally {
             artifacts.cleanup();
+            // increment the cycle count for a producer with primary status for both cases of cycle runs - success or failure.
```

Comment:
Just a very minor note on code style- comments like this one and the one on line 341 are a bit superfluous, so the added verbosity feels a little less clean. For e.g. here it is pretty clear from a quick glance that if an action is performed in a finally block it is invoked regardless of success or failure, and for line 341 again a quick glance reveals that if instance is not primary then we are doing that action.

Reply:
Ack. Noted.

Reply:
I think I was going after explaining the increment condition and it would have made more sense in the previous commit when increment was only being done for successful cycles.